daemon/config_windows.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/spf13/pflag"
 )
 
+const (
+	// stockRuntimeName is the reserved name/alias used to represent the
+	// OCI runtime being shipped with the docker daemon package.
+	stockRuntimeName = "runc"
```

Comment:
@jhowardmsft : this change was introduced because the stock runtime was being initialized to runc for all platforms and we wanted it to be runz for Solaris. Due to changes that have come about since then this is no longer required, and the logic can go back to what it was.
I'll revert this change.